tests/unit/repo/test_repo.py

```diff
@@ -89,15 +89,13 @@ def test_skip_graph_checks(tmp_dir, dvc, mocker, run_copy):
     mock_build_graph.reset_mock()
     dvc._skip_graph_checks = True
     tmp_dir.gen("baz", "baz text")
-    dvc.add("baz")
```

Comment:
`dvc.add()` requires building graph, so the optimization that we had for not collecting all the `.dvc` files in the workspace no longer works.

Reply:
That seems like an odd side effect of this change. I would not expect that behaviour to change because of this PR.

Reply:
`find_outs_by_path` has to collect all `.dvc` files.